Pearson-Advance/frontend-app-institution-portal
# PADV-1043 - Course Details button courses workflow

### Ticket
https://agile-jira.pearson.com/browse/PADV-1043

### Description
Add course details button in classes table for courses workflow.

### Changes made
- Add course details button.

### Screenshots
- Course details button (Figma)

![image](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/73655023/9d815a0f-24b0-494e-a0cd-4db27eaab4df)



- Course details button

![image](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/73655023/3c22ca2e-2c36-4b02-a9e5-d0c6e3f7b5d7)






### How to test

- Clone the Institution Portal MFE.
- Go to vue/PADV-1043 branch.
- Run npm install.
- Run npm run start.
- Go to http://localhost:1980/courses.
- Click over a course name.
- Verify if the button is renderized as expected.
- Click the button and verify if it redirect you to Learning microfrontend.
